### PR TITLE
🚀 Release v1.7.6

### PR DESCRIPTION
**Pull request by chandankumar4**
**What this PR does / why we need it**:
- Release note for v1.7.6

/area release

### CODE REVIEW

File: CHANGELOG/v1.7.6.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.5
+## :chart_with_upwards_trend: Overview
+- 3 new commits merged
+
+## :seedling: Others
```
Comment (sbueringer) [inline CHANGELOG/v1.7.6.md:12]: Let's also add https://github.com/kubernetes-sigs/cluster-api/pull/11159 (just merged today)

Comment (chandankumar4) [reply CHANGELOG/v1.7.6.md:12]: Added

**Comment by Sunnatillo** [comment]
/lgtm
/hold

I will unhold 30 mins before release cut meeting.

**Comment by sbueringer** [comment]
Thx!

/approve

**Pull request merged**